docs/Manuals/Server/fcdb.rst

```diff
@@ -46,9 +46,11 @@ storing passwords with MD5 hashes.
 
 .. code-block:: sh
 
-    $ sudo apt install luarocks
-    $ sudo luarocks --lua-version 5.3 install md5
-    $ sudo luarocks --lua-version 5.3 install luasql-sqlite3
+    $ sudo apt install luarocks lua5.3 liblua5.3-dev libsqlite3-dev
+    $ lua -v  # output should be 5.3.x
```

Comment:
This will not necessarily be true if the user changed the default Lua version via `update-alternatives` (do `update-alternatives --display lua-interpreter`). Lua 5.3 can be invoked using `lua5.3`, but then the check becomes tautological.
I would suggest to drop this check: Freeciv21 is currently always compiled against Lua 5.3.